src/components/Admin/Messenger.jsx

```diff
@@ -7,90 +7,76 @@ import Input from "./Input";
 import Textarea from "./Textarea";
 import Button from "./Button";
 import toast from "react-hot-toast";
-
-const ADDRESSEES = [
-  {
-    uid: "1",
-    name: "Menthy Wu",
-    email: "yhung022@ucr.edu",
-    team: "b",
-    major: "Computer Science",
-    status: "pending",
-    selected: false,
-  },
-  {
-    uid: "2",
-    name: "Divyank Shah",
-    email: "yhung022@ucr.edu",
-    team: "c",
-    major: "Computer Science",
-    status: "accepted",
-    selected: false,
-  },
-  {
-    uid: "3",
-    name: "Shing Hung",
-    email: "yhung022@ucr.edu",
-    team: "d",
-    major: "Computer Science",
-    status: "rejected",
-    selected: false,
-  },
-];
+import Upload from "./Upload";
+import { userList } from "@/data/mock/messengerUsers";
+import { messengerFilters } from "@/data/Filters";
 
 const Messenger = () => {
-  const [subjectText, setSubjectText] = useState("");
-  const [messageBody, setMessageBody] = useState("");
-  const [filters, setFilters] = useState({
-    accepted: false,
-    rejected: false,
-    pending: false,
-    volunteers: false,
-    mentors: false,
-    judges: false,
+  const [email, setEmail] = useState({
+    sendto: [],
+    subject: "",
+    body: "",
+    files: [],
   });
-  const [filteredParticipants, setfilteredParticipants] = useState(ADDRESSEES);
+  const [filters, setFilters] = useState(messengerFilters);
+  const [user, setUser] = useState(userList);
```

Comment:
don't need a user state, you will be getting ur data in an array, we will do processing in backend for participants -> participants.emails